spec/system/referee_interface/referee_does_not_respond_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'Referee does not respond within 5 days', sidekiq: true do
+  include CandidateHelper
+
+  scenario 'A chase email is sent if a referee does not respond within 5 days' do
+    FeatureFlag.activate('training_with_a_disability')
+
+    given_a_candidate_completed_an_application
+    when_the_candidate_submits_the_application
+    and_i_do_not_respond_within_5_days
+    then_i_am_sent_a_chase_email
```

Comment:
Perhaps `and_the_referee_does_not_respond_within_5_days` (we haven't explained that we're a referee)